pkg/ingester/ingester.go

```diff
@@ -205,14 +209,22 @@ func New(cfg Config, clientConfig client.Config, limits *validation.Overrides, c
 	}
 
 	var err error
-	i.lifecycler, err = ring.NewLifecycler(cfg.LifecyclerConfig, i, "ingester")
+	i.limiter = NewSeriesLimiter(limits, i.lifecycler, cfg.LifecyclerConfig.RingConfig.ReplicationFactor, cfg.ShardByAllLabels)
```

Comment:
`i.lifecycler` has not been created yet. You have to initialize the limiter faster the lifecycler is initialized (take a look at the current implementation in `master` for details).